### DISCUSSION TITLE
Is it possible to bias the decoder toward a small set of tokens?

### DISCUSSION THREAD

**Discussion by tilsen** [openai/whisper#1679] [Q&A] [answered]
For example, if I wanted to transcribe audio in which there is a small set of words, is it possible to bias the decoder model to prefer those words over similar ones, so that transcriptions become more likely to output those words?

**Comment by glangford** [comment, marked as answer]
Try the `--initial_prompt` option, and fyi below.

* https://github.com/openai/whisper/discussions/66
* https://github.com/openai/whisper/discussions/1159
* https://github.com/BriansIDP/WhisperBiasing
